### PR TITLE
PatientSex Updated and migrations file generated

### PR DESCRIPTION
**Pull request by cornelia247**
PatientSex Updated and migrations file generated for Issue https://github.com/FNNDSC/ChRIS_ultron_backEnd/issues/433

### CODE REVIEW

File: chris_backend/pacsfiles/models.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class PACSFile(models.Model):
     PatientName = models.CharField(max_length=150, blank=True)
     PatientBirthDate = models.DateField(blank=True, null=True)
     PatientAge = models.IntegerField(blank=True, null=True)
-    PatientSex = models.CharField(max_length=1, choices=[('M', 'Male'), ('F', 'Female')],
+    PatientSex = models.CharField(max_length=2, choices=[('M', 'Male'), ('F', 'Female'), ('O', 'Other')],
```
Comment (jennydaman) [inline chris_backend/pacsfiles/models.py:23]: why did you change `max_length` to 2?

**Comment by jennydaman** [comment]
Also towards the end of our meeting we discussed unit tests — it would be awesome if you could either add a test where `PatientSex=O`, or change an existing test to cover this case.

**Comment by cornelia247** [comment]
A typo probably when it was still "NB", yeah I did think about creating one, I'll get on it now

**Review by jennydaman — APPROVED** [review]
